dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -18,6 +18,15 @@ import 'package:path/path.dart' as path;
 /// adding Flutter to an existing iOS app.
 Future<void> main() async {
   await task(() async {
+    // Update pod repo.
+    await eval(
+      'pod',
+      <String>['repo', 'update'],
+      environment: <String, String>{
+        'LANG': 'en_US.UTF-8',
```

Comment:
Out of curiosity, what kind of bad things could happen if we don't override the LANG?

Reply:
A harmless warning  #45710 https://github.com/CocoaPods/CocoaPods/issues/9347
The tool does this
https://github.com/flutter/flutter/blob/8074811c2f1ec801aff9487e4ba7f49ab540bbd3/packages/flutter_tools/lib/src/macos/cocoapods.dart#L121
but this is invoked outside the tool.  The fewer avoidable warnings in the logs the better, imo.

Reply:
Never mind, I fixed this at some point, `exec`, `eval` are getting `LANG` added later: #82308